src/MotionControl.c

```diff
@@ -1287,7 +1287,7 @@ BOOL Ros_MotionControl_StartMotionMode(MOTION_MODE mode)
     {
         //TODO(gavanderhoorn): special check for "job is not loaded"
         Ros_Debug_BroadcastMsg(
-            "Can't start '%s' because: 0x%04X", g_nodeConfigSettings.inform_job_name, rData.err_no);
+            "Can't start '%s' because: 0x%04X %s", g_nodeConfigSettings.inform_job_name, rData.err_no, Ros_ErrorHandling_Only_ErrNo_ToString(rData.err_no));
```

Comment:
please see my https://github.com/Yaskawa-Global/motoros2/issues/102#issuecomment-1661579552.

I would suggest we replace the other case where we print `rData.err_no` directly as well.